.travis.yml

```diff
@@ -25,9 +26,6 @@ git:
 #before_script: # homebrew for mac
 #  - if [ $TRAVIS_OS_NAME = osx ]; then brew install gcc; fi
 
-# uncomment the following lines to override the default test script
-script:
-  - julia -e 'Pkg.clone(pwd()); Pkg.build("Setfield"); VERSION < v"0.7-" && Pkg.add("StaticArrays"); Pkg.test("Setfield"; coverage=true)'
```

Comment:
So for Julia 1.0 we need to remove this since Pkg is not imported by default anymore.  Besides, we don't need to manually install it since we already have it in `test/REQUIRE`.